scripts/releaseFramework.js

```diff
@@ -10,34 +12,39 @@ const child_process = require('child_process');
 const fetch = require('node-fetch');
 const path = require('path');
 
-let version = child_process.execSync('git rev-parse --short HEAD').toString().trim();
+let version = child_process
+  .execSync('git rev-parse --short HEAD')
+  .toString()
+  .trim();
 
 (async () => {
   /**
    * Create Github release that points to the latest commit
    */
-  const release = await (
-    await fetch(
-      `https://api.github.com/repos/kiwicom/mobile/releases?access_token=${process.env.GITHUB_TOKEN}`,
-      {
-        method: 'POST',
-        headers: {
-          Accept: 'application/json',
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({
-          tag_name: version,
-          target_commitish: 'master',
-          name: version,
-          draft: false,
-          prerelease: false,
-        }),
-      }
-    )
-  ).json();
+  const release = await (await fetch(
+    `https://api.github.com/repos/kiwicom/mobile/releases?access_token=${
+      // $FlowFixMe env.GITHUB_TOKEN will be a string
```

Comment:
Fix it directly please - there is a reason for this Flow error. See for example: https://github.com/facebook/flow/issues/1192

Reply:
Ok once I have the PR actually working (WIP) I'll do that.